### PR TITLE
Replace internal RxSharedPreferences with library

### PR DESCRIPTION
**Pull request by steffandroid**

### CODE REVIEW

File: src/internalDebug/java/com/jakewharton/u2020/data/prefs/RxSharedPreferences.java [hunk 1]
```diff
@@ -1,111 +0,0 @@
...
-import rx.Subscription;
-import rx.functions.Action0;
-import rx.functions.Action1;
-import rx.functions.Func1;
-import rx.subscriptions.Subscriptions;
-
-import static android.content.SharedPreferences.OnSharedPreferenceChangeListener;
-import static com.jakewharton.u2020.util.Preconditions.checkNotNull;
-
-public final class RxSharedPreferences {
```
Comment (JakeWharton) [inline src/internalDebug/java/com/jakewharton/u2020/data/prefs/RxSharedPreferences.java]: Wow. I forgot I wrote this.

**Comment by JakeWharton** [comment]
I kicked the CI build again to see if it was a flake.

File: src/internalDebug/java/com/jakewharton/u2020/data/prefs/RxSharedPreferences.java (hunk 1 above)
Comment (steffandroid) [inline src/internalDebug/java/com/jakewharton/u2020/data/prefs/RxSharedPreferences.java]: I've just realised that this class still exists over in [RxBinding](https://github.com/JakeWharton/RxBinding/blob/98303953b83e3001680ee8d8c1726580e219666c/rxandroid/src/main/java/rx/android/content/RxSharedPreferences.java).

**Comment by JakeWharton** [comment]
Thanks!

**Pull request merged**